enterprise/server/remote_execution/platform/platform.go

```diff
@@ -59,6 +60,10 @@ const (
 	// Using the property defined here: https://github.com/bazelbuild/bazel-toolchains/blob/v5.1.0/rules/exec_properties/exec_properties.bzl#L164
 	dockerRunAsRootPropertyName = "dockerRunAsRoot"
 
+	// A BuildBuddy Compute Unit is defined as 1 cpu and 2.5GB of memory.
+	EstimatedComputeUnitsPropertyName = "EstimatedComputeUnits"
+	EstimatedFreeDiskPropertyName     = "EstimatedFreeDiskBytes"
```

Comment:
I don't follow what this is for based on the name. (Why would a client need to tell us the estimated free disk of a task?)

Can you add a comment?

Reply:
@siggisim do you know the context for the name of this prop?

Reply:
I think this was added by @vadimberezniker in this PR https://github.com/buildbuddy-io/buildbuddy/commit/0536ec1548e377a815bd7a5f62e4480428a90176#diff-9b6312e0c353cc319b101e1cb71608ed26be804af9e31d0b1c6d5ebe5552992eR16

Reply:
This is used to specify how much "scratch space" beyond the input files a task requires to be able to function.
I added it this to support Managed Bazel + Firecracker because for Firecracker we need to decide ahead of time how big the workspace filesystem is going to be and Managed Bazel requires a relatively large amount of free space compared to typical actions.

Reply:
SG, @bduffany can you add a comment to the property const?

Reply:
@tylerwilliams Done